### PR TITLE
[TR][SLA] PIM-4748: remove left joins to fix perf issue with many locales

### PR DESCRIPTION
**Pull request by nidup**
| Q                  | A
| ------------------ | ---
| Migration script?  | N
| Behats             | Y
| Specs & Static     | Y
| Changelog updated  | Y

### CODE REVIEW

File: features/Context/FixturesContext.php [hunk 1]
```diff
@@ -41,6 +41,11 @@ class FixturesContext extends RawMinkContext
         'english' => 'en_US',
         'french'  => 'fr_FR',
         'german'  => 'de_DE',
+        'spanish' => 'es_ES',
```
Comment (sumbobyboys) [inline features/Context/FixturesContext.php]: Could you align ?

File: spec/Pim/Bundle/CatalogBundle/Doctrine/ORM/ProductRepositorySpec.php [hunk 2]
```diff
@@ -2,20 +2,73 @@
...
+
+    function it_does_join_product_attribute_and_values_when_find_one_product(
+        $em,
+        $pqbFactory,
+        QueryBuilder $queryBuilder,
+        Expr $expr,
+        AbstractQuery $query,
+        ProductQueryBuilder $pqb
+    ) {
+        $pqbFactory->create()->willReturn($pqb);
```
Comment (sumbobyboys) [inline spec/Pim/Bundle/CatalogBundle/Doctrine/ORM/ProductRepositorySpec.php]: l'enfer de mode

**Comment by phaseinducer** [comment]
Strange to use a behat scenario to test perf issues : /

**Comment by nidup** [comment]
@phaseinducer we would add a dedicated perf build on the CI, for now, i'm not too disturbed to have a behat which use a lot of locales for a product as it is a real customer use case

**Pull request merged**